src/components/ProfileValidation/Attestation.tsx

```diff
@@ -1,74 +1,86 @@
-
-import { Box } from '@mui/material'
+import { Box, Button } from '@mui/material'
```

Comment:
- design changes
 - moved continue button and enabled/disabled logic into component